packages/appstore/src/components/currency-switcher-card/real/real-account-card.tsx

```diff
@@ -34,7 +34,7 @@ const RealAccountCard = () => {
                     {getCurrencyName(currency)}
                 </Text>
             }
-            icon={currency}
+            icon={IsIconCurrency(currency) ? currency : 'USD'}
```

Comment:
Duplicated code here and in line 59.
better to call a function then e.g. `getDefaultCurrency`